core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/filter/FilterPredicate.java

```diff
@@ -19,7 +19,7 @@
     @JsonProperty(value = "condition", required = true)
     public ComparisonType condition;
 
-    @JsonProperty(value = "value", required = true)
+    @JsonProperty(value = "value")
```

Comment:
now the value is no longer required, we need to add some checks to prevent it being empty for comparison cases, can you add the validation code in `operatorExecutor` function under `SpecializedFilterOpDesc`? If the value is null/empty for comparison types, throw an exception.